Apromore-Extras/APMLogModule/src/main/java/org/apromore/apmlog/filter/typefilters/CaseSectionAttributeCombinationFilter.java

```diff
@@ -46,9 +46,8 @@ public static boolean toKeep(ATrace trace, LogFilterRule logFilterRule) {
 
     private static boolean conformRule(ATrace trace, LogFilterRule logFilterRule) {
 
-
         Set<String> primaryValues = logFilterRule.getPrimaryValuesInString();
```

Comment:
Might be best to rename this `primaryValuesInString` since there's also `logFilterRule.getPrimaryValues()` and it's even being used in the next few lines.  It'd be easy to mistakenly assume that `primaryValues` and `secondaryValues` work similarly to one another.